composer.json

```diff
@@ -53,6 +53,7 @@
     },
     "conflict": {
         "doctrine/dbal": "<2.6.0",
+        "doctrine/mongodb-odm": "<2.0",
```

Comment:
```suggestion
        "doctrine/mongodb-odm": "<2.2",
```

Reply:
This should match the require-dev dependency

Reply:
Done.